2025-09-19 10:02:45,811	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 5263
2025-09-19 11:11:22,653	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 11:11:22,972	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 1146
2025-09-19 11:11:23,226	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/users/@me, 错误代码: 401, 返回内容: {'message': '接口访问源IP不在白名单', 'code': 11298, 'err_code': 40023002, 'trace_id': '3c2de57bea0faaf8d3de6f4b7c97e4c7'}, trace_id:3c2de57bea0faaf8d3de6f4b7c97e4c7
2025-09-19 11:16:04,840	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 11:16:05,106	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 864
2025-09-19 11:16:05,412	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/users/@me, 错误代码: 401, 返回内容: {'message': '接口访问源IP不在白名单', 'code': 11298, 'err_code': 40023002, 'trace_id': '103672c39a425fa4d9e97d3fcb111de5'}, trace_id:103672c39a425fa4d9e97d3fcb111de5
2025-09-19 11:17:09,493	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 11:17:09,695	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 799
2025-09-19 11:17:09,948	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/users/@me, 错误代码: 401, 返回内容: {'message': '接口访问源IP不在白名单', 'code': 11298, 'err_code': 40023002, 'trace_id': '5aa425a86197e84d3e6fe8a249f357a5'}, trace_id:5aa425a86197e84d3e6fe8a249f357a5
2025-09-19 11:21:18,989	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 11:21:19,204	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 550
2025-09-19 11:21:19,450	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/users/@me, 错误代码: 401, 返回内容: {'message': '接口访问源IP不在白名单', 'code': 11298, 'err_code': 40023002, 'trace_id': '6b1c100468ddde0065d8479d2bb21c1a'}, trace_id:6b1c100468ddde0065d8479d2bb21c1a
2025-09-19 11:25:19,349	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 11:25:19,577	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 309
2025-09-19 11:25:20,515	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-09-19 11:25:20,516	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-09-19 11:25:20,517	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-09-19 11:25:20,518	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-09-19 11:25:20,713	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-09-19 11:25:20,839	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「七秒钟的记忆-测试中」启动成功！
2025-09-19 11:25:20,841	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-09-19 11:28:31,174	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 11:28:31,411	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 117
2025-09-19 11:28:31,920	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-09-19 11:28:31,921	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-09-19 11:28:31,921	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-09-19 11:28:31,921	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-09-19 11:28:32,085	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-09-19 11:28:32,218	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「七秒钟的记忆-测试中」启动成功！
2025-09-19 11:28:32,219	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-09-19 11:29:04,008	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 11:29:04,211	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 85
2025-09-19 11:29:04,738	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-09-19 11:29:04,739	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-09-19 11:29:04,740	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-09-19 11:29:04,740	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-09-19 11:29:04,928	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-09-19 11:29:05,049	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「七秒钟的记忆-测试中」启动成功！
2025-09-19 11:29:05,050	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-09-19 11:29:27,660	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 11:29:27,845	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 61
2025-09-19 11:29:28,345	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-09-19 11:29:28,345	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-09-19 11:29:28,346	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-09-19 11:29:28,346	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-09-19 11:29:28,564	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-09-19 11:29:28,665	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「七秒钟的记忆-测试中」启动成功！
2025-09-19 11:29:28,666	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-09-19 11:36:14,163	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 11:36:14,424	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 7200
2025-09-19 11:36:16,372	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-09-19 11:36:16,373	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-09-19 11:36:16,374	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-09-19 11:36:16,375	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-09-19 11:36:16,631	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-09-19 11:36:16,772	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「七秒钟的记忆-测试中」启动成功！
2025-09-19 11:36:16,773	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-09-19 11:42:30,038	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 11:42:30,437	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 6824
2025-09-19 11:42:31,221	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-09-19 11:42:31,221	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-09-19 11:42:31,222	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-09-19 11:42:31,223	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-09-19 11:42:31,401	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-09-19 11:42:31,528	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「七秒钟的记忆-测试中」启动成功！
2025-09-19 11:42:31,529	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-09-19 11:46:40,864	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 11:46:41,067	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 6573
2025-09-19 11:46:42,102	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-09-19 11:46:42,102	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-09-19 11:46:42,102	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-09-19 11:46:42,103	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-09-19 11:46:42,253	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-09-19 11:46:42,353	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「七秒钟的记忆-测试中」启动成功！
2025-09-19 11:46:42,353	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-09-19 12:31:33,088	[INFO]	(gateway.py:56)on_closed	[botpy] 关闭, 返回码: 1006, 返回信息: None
2025-09-19 12:31:33,089	[INFO]	(gateway.py:134)ws_connect	[botpy] ws关闭, 停止接收消息!
2025-09-19 12:31:38,100	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-09-19 12:31:38,100	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-09-19 12:31:38,101	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-09-19 12:31:38,384	[INFO]	(gateway.py:175)ws_resume	[botpy] 重连启动...
2025-09-19 12:31:38,507	[INFO]	(gateway.py:90)on_message	[botpy] 机器人重连成功! 
2025-09-19 12:31:38,508	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-09-19 12:53:18,188	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 12:53:18,798	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 2575
2025-09-19 12:53:20,394	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-09-19 12:53:20,395	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-09-19 12:53:20,395	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-09-19 12:53:20,395	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-09-19 12:53:21,223	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-09-19 12:53:21,526	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「七秒钟的记忆-测试中」启动成功！
2025-09-19 12:53:21,526	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-09-19 12:59:01,172	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 12:59:01,466	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 2233
2025-09-19 12:59:02,204	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-09-19 12:59:02,205	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-09-19 12:59:02,206	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-09-19 12:59:02,207	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-09-19 12:59:02,464	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-09-19 12:59:02,626	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「七秒钟的记忆-测试中」启动成功！
2025-09-19 12:59:02,626	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-09-19 13:02:44,772	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 13:02:45,259	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 2009
2025-09-19 13:02:45,930	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-09-19 13:02:45,931	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-09-19 13:02:45,931	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-09-19 13:02:45,931	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-09-19 13:02:46,442	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-09-19 13:02:46,613	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「七秒钟的记忆-测试中」启动成功！
2025-09-19 13:02:46,614	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-09-19 13:22:47,499	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 13:22:47,711	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 806
2025-09-19 13:22:48,875	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-09-19 13:22:48,877	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-09-19 13:22:48,879	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-09-19 13:22:48,879	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-09-19 13:22:49,078	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-09-19 13:22:49,214	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「七秒钟的记忆-测试中」启动成功！
2025-09-19 13:22:49,215	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-09-19 13:24:40,091	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 13:24:40,663	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 694
2025-09-19 13:24:41,940	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-09-19 13:24:41,941	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-09-19 13:24:41,941	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-09-19 13:24:41,942	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-09-19 13:24:42,117	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-09-19 13:24:42,243	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「七秒钟的记忆-测试中」启动成功！
2025-09-19 13:24:42,243	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-09-19 13:27:05,594	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 13:27:05,962	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 548
2025-09-19 13:27:07,137	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-09-19 13:27:07,137	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-09-19 13:27:07,138	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-09-19 13:27:07,138	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-09-19 13:27:07,553	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-09-19 13:27:07,694	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「七秒钟的记忆-测试中」启动成功！
2025-09-19 13:27:07,694	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-09-19 13:28:29,813	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 13:28:30,175	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 464
2025-09-19 13:28:31,021	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-09-19 13:28:31,022	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-09-19 13:28:31,022	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-09-19 13:28:31,023	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-09-19 13:28:31,501	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-09-19 13:28:31,642	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「七秒钟的记忆-测试中」启动成功！
2025-09-19 13:28:31,642	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-09-19 13:29:10,082	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://sandbox.api.sgroup.qq.com/v2/groups/DAA3AA5476808C1803B3FD5279ED9AB6/messages, 错误代码: 500, 返回内容: {'message': 'invalid content', 'code': 304061, 'err_code': 304061, 'trace_id': 'f5519ef8141775989881efd1bddc2f7f'}, trace_id:f5519ef8141775989881efd1bddc2f7f
2025-09-19 13:29:27,796	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 13:29:28,175	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 406
2025-09-19 13:29:28,945	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-09-19 13:29:28,945	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-09-19 13:29:28,945	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-09-19 13:29:28,945	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-09-19 13:29:29,144	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-09-19 13:29:29,247	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「七秒钟的记忆-测试中」启动成功！
2025-09-19 13:29:29,248	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-09-19 13:29:37,587	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://sandbox.api.sgroup.qq.com/v2/groups/DAA3AA5476808C1803B3FD5279ED9AB6/messages, 错误代码: 400, 返回内容: {'message': '主动消息失败, 无权限', 'code': 40034102, 'err_code': 40034102, 'trace_id': '9db07b5f2118cb9d160e643efe274523'}, trace_id:9db07b5f2118cb9d160e643efe274523
2025-09-19 13:31:08,586	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 13:31:08,895	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 305
2025-09-19 13:31:09,572	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-09-19 13:31:09,573	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-09-19 13:31:09,574	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-09-19 13:31:09,575	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-09-19 13:31:09,733	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-09-19 13:31:09,841	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「七秒钟的记忆-测试中」启动成功！
2025-09-19 13:31:09,844	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-09-19 13:32:07,608	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 13:32:07,803	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 246
2025-09-19 13:32:08,414	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-09-19 13:32:08,416	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-09-19 13:32:08,417	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-09-19 13:32:08,418	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-09-19 13:32:08,590	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-09-19 13:32:08,715	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「七秒钟的记忆-测试中」启动成功！
2025-09-19 13:32:08,716	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-09-19 13:32:14,553	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://sandbox.api.sgroup.qq.com/v2/groups/DAA3AA5476808C1803B3FD5279ED9AB6/messages, 错误代码: 500, 返回内容: {'message': 'invalid content', 'code': 304061, 'err_code': 304061, 'trace_id': '0188749dffe38115f086f02829341f16'}, trace_id:0188749dffe38115f086f02829341f16
2025-09-19 13:34:00,313	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 13:34:00,735	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 134
2025-09-19 13:34:01,635	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-09-19 13:34:01,638	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-09-19 13:34:01,639	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-09-19 13:34:01,640	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-09-19 13:34:01,941	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-09-19 13:34:02,093	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「七秒钟的记忆-测试中」启动成功！
2025-09-19 13:34:02,094	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-09-19 13:34:34,695	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 13:34:35,145	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 99
2025-09-19 13:34:35,687	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-09-19 13:34:35,688	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-09-19 13:34:35,689	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-09-19 13:34:35,689	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-09-19 13:34:35,891	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-09-19 13:34:36,007	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「七秒钟的记忆-测试中」启动成功！
2025-09-19 13:34:36,007	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-09-19 13:34:41,140	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://sandbox.api.sgroup.qq.com/v2/groups/DAA3AA5476808C1803B3FD5279ED9AB6/messages, 错误代码: 400, 返回内容: {'message': '主动消息失败, 无权限', 'code': 40034102, 'err_code': 40034102, 'trace_id': '71fa78dc7683f7e7338597a6e3690736'}, trace_id:71fa78dc7683f7e7338597a6e3690736
2025-09-19 13:34:41,154	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://sandbox.api.sgroup.qq.com/v2/groups/DAA3AA5476808C1803B3FD5279ED9AB6/messages, 错误代码: 500, 返回内容: {'message': 'invalid content', 'code': 304061, 'err_code': 304061, 'trace_id': 'bfd6b378c97f118011c318307ca2db1d'}, trace_id:bfd6b378c97f118011c318307ca2db1d
2025-09-19 13:35:26,583	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 13:35:26,866	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 7200
2025-09-19 13:35:28,060	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-09-19 13:35:28,060	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-09-19 13:35:28,060	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-09-19 13:35:28,060	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-09-19 13:35:28,273	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-09-19 13:35:28,384	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「七秒钟的记忆-测试中」启动成功！
2025-09-19 13:35:28,384	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-09-19 13:35:31,435	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://sandbox.api.sgroup.qq.com/v2/groups/DAA3AA5476808C1803B3FD5279ED9AB6/messages, 错误代码: 400, 返回内容: {'message': '主动消息失败, 无权限', 'code': 40034102, 'err_code': 40034102, 'trace_id': '76482f60efae122e0f1ef7cddf3f6364'}, trace_id:76482f60efae122e0f1ef7cddf3f6364
2025-09-19 13:37:03,897	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 13:37:04,257	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 7103
2025-09-19 13:37:05,185	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-09-19 13:37:05,186	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-09-19 13:37:05,188	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-09-19 13:37:05,190	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-09-19 13:37:05,481	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-09-19 13:37:05,657	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「七秒钟的记忆-测试中」启动成功！
2025-09-19 13:37:05,658	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-09-19 13:39:08,315	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 13:39:09,196	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 6978
2025-09-19 13:39:10,762	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-09-19 13:39:10,764	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-09-19 13:39:10,765	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-09-19 13:39:10,766	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-09-19 13:39:11,341	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-09-19 13:39:11,564	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「七秒钟的记忆-测试中」启动成功！
2025-09-19 13:39:11,565	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-09-19 13:39:54,057	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://sandbox.api.sgroup.qq.com/v2/groups/DAA3AA5476808C1803B3FD5279ED9AB6/messages, 错误代码: 400, 返回内容: {'message': '主动消息失败, 无权限', 'code': 40034102, 'err_code': 40034102, 'trace_id': 'e6fab7396060fbbf31cf084addb3af2d'}, trace_id:e6fab7396060fbbf31cf084addb3af2d
2025-09-19 13:40:04,781	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://sandbox.api.sgroup.qq.com/v2/groups/DAA3AA5476808C1803B3FD5279ED9AB6/messages, 错误代码: 400, 返回内容: {'message': '主动消息失败, 无权限', 'code': 40034102, 'err_code': 40034102, 'trace_id': '92d1d38fddca7d20a9dc988dd1104852'}, trace_id:92d1d38fddca7d20a9dc988dd1104852
2025-09-19 16:31:02,870	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-09-19 16:31:03,278	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 6459
2025-09-19 16:31:03,994	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-09-19 16:31:03,994	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-09-19 16:31:03,995	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-09-19 16:31:03,995	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-09-19 16:31:04,212	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-09-19 16:31:04,369	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「七秒钟的记忆-测试中」启动成功！
2025-09-19 16:31:04,369	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
